### ISSUE TITLE
Users receiving email notifications even though they are disabled in settings

### ISSUE THREAD

**Issue by milosdelsystems** [grokability/snipe-it#15932]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

I'm running snipeit on latest version (v7.1.15).
I have unchecked options to send checkin/checkout emails to users in the Category settings, however, users are still getting emails each time asset is checked out or checked in to them.
Also this is only happening when: Require users to confirm acceptance of assets in this category is turned on in Category Settings

### Reproduction steps

1. Require users to confirm acceptance of assets in this category is turned on
2. checkin/checkout e-mail notifications are turned off in asset category settings
3. checkin asset from user
4. user recieves e-mail, even though it shouldn't

### Expected behavior

When i checkin/checkout asset from user, the user shouldn't receive an email

### Screenshots

<img width="715" alt="Screenshot 2024-12-09 at 09 44 45" src="https://github.com/user-attachments/assets/c3a917ab-7a36-4ff8-a3b0-7b5c341e27ad">

I have the option 'send email to user on check-in' turned off for assets(as shown in the screenshots above). However, users still receive emails when I check in accessories from them.

### Snipe-IT Version

v7.1.15

### Operating System

linux debian 12

### Web Server

nginx 

### PHP Version

8.3

**Comment by milosdelsystems** [comment]
The same thing is happening with accessories as well

**Comment by swift2512** [comment]
That's the way it is since forever. The only way is to leave mail settings blank, but then you won't get acceptance alerts for admin accounts.
Also, I encourage you to register all your items as assets, because you can't generate reports for accessories or see which items haven't been accepted yet.

**Comment by milosdelsystems** [comment]
No that's not true,
I have one more server with the same settings, only difference is that snipeit is running on version v7.0.9 , and users don't receive e-mails when i checkin assets/accessories from them even though Require users to confirm acceptance of assets in this category is turned on

**Comment by milosdelsystems** [comment]
I had the same issue in the version v6.3.0, as you can see in this bug report, and that has been fixed, but now it has returned
https://github.com/snipe/snipe-it/issues/14237

**Comment by swift2512** [comment]
You have enabled "require user to confirm acceptance". This triggers confirmation emails. I've noticed this on one of my installs maybe 4-6 months ago and considered it as a feature.
Even on asset check-out page you are shown message that email will be sent to user. (If that item has acceptance enabled.)

**Comment by milosdelsystems** [comment]
Yes, and that's okey, but if i checkin asset from user (meaning it's no longer associated with the user and is available for other users to be assigned) , the user still recieves e-mail, which should not happen, even when "require user to confirm acceptance" is turned on.

**Comment by swift2512** [comment]
Can confirm this behaviour on my 7.1.15.
Can't see how it's a problem. User gets acceptance email and then is once again informed that item he has accepted before, was taken from him.

**Comment by milosdelsystems** [comment]
This was not the case before, I want to have control over whether the user receives an email upon check-in

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by milosdelsystems** [comment]
Issue is not present anymore on version v7.1.15
